liquibase-service/src/main/resources/db/rdb_modern/routines/013-sp_hepatitis_datamart_postprocessing-003.sql

```diff
@@ -0,0 +1,2609 @@
+
+
+        INSERT INTO [DBO].[JOB_FLOW_LOG]( BATCH_ID, [DATAFLOW_NAME], [PACKAGE_NAME], [STATUS_TYPE], [STEP_NUMBER], [STEP_NAME], [ROW_COUNT] )
+        VALUES( @BATCH_ID, 'HEPATITIS_DATAMART', 'Hepatitis_Case_DATAMART', 'START', @PROC_STEP_NO, @PROC_STEP_NAME, @ROWCOUNT_NO );
+
+
+        COMMIT TRANSACTION;
+
+
+---------------------------------------------------17. CREATE TABLE TMP_HEP_PAT_PROV ---------------------------
+        BEGIN TRANSACTION;
+
+        SET @Proc_Step_name = 'Generating  #TMP_HEP_PAT_PROV';
+        SET @Proc_Step_no = 17;
+
+        IF OBJECT_ID('#TMP_HEP_PAT_PROV', 'U') IS NOT NULL
+            BEGIN
+                DROP TABLE TMP_HEP_PAT_PROV;
+
+            END;
+
+        SELECT DISTINCT
+            TMP_F_PAGE_CASE.INVESTIGATION_KEY AS HEP_PAT_PROV_INV_KEY, P.PROVIDER_LOCAL_ID, P.PROVIDER_FIRST_NAME AS PHYSICIAN_FIRST_NM, P.PROVIDER_MIDDLE_NAME AS PHYSICIAN_MIDDLE_NM, P.PROVIDER_LAST_NAME AS PHYSICIAN_LAST_NM, CAST(NULL AS varchar(300)) AS PHYS_NAME, P.PROVIDER_CITY AS PHYS_CITY, P.PROVIDER_STATE AS PHYS_STATE, P.PROVIDER_COUNTY AS PHYS_COUNTY, CAST(NULL AS varchar(300)) AS PHYSICIAN_ADDRESS_USE_DESC, CAST(NULL AS varchar(300)) AS PHYSICIAN_ADDRESS_TYPE_DESC, P.PROVIDER_ADD_TIME, P.PROVIDER_LAST_CHANGE_TIME, P.PROVIDER_UID AS PHYSICIAN_UID, INVGTR.PROVIDER_FIRST_NAME AS INVESTIGATOR_FIRST_NM, INVGTR.PROVIDER_MIDDLE_NAME AS INVESTIGATOR_MIDDLE_NM, INVGTR.PROVIDER_LAST_NAME AS INVESTIGATOR_LAST_NM, CAST(NULL AS varchar(300)) AS INVESTIGATOR_NAME, INVGTR.PROVIDER_UID AS INVESTIGATOR_UID, REPTORG.ORGANIZATION_NAME AS RPT_SRC_SOURCE_NM, REPTORG.ORGANIZATION_COUNTY_CODE AS RPT_SRC_COUNTY_CD, REPTORG.ORGANIZATION_COUNTY AS RPT_SRC_COUNTY, REPTORG.ORGANIZATION_CITY AS RPT_SRC_CITY, REPTORG.ORGANIZATION_STATE AS RPT_SRC_STATE, CAST(NULL AS varchar(300)) AS REPORTING_SOURCE_ADDRESS_USE, CAST(NULL AS varchar(300)) AS REPORTING_SOURCE_ADDRESS_TYPE, REPTORG.ORGANIZATION_UID AS REPORTING_SOURCE_UID
+        INTO #TMP_HEP_PAT_PROV
+        FROM dbo.F_PAGE_CASE AS PAGE_CASE WITH(NOLOCK)
+                 INNER JOIN
+             #TMP_F_PAGE_CASE TMP_F_PAGE_CASE
+             ON TMP_F_PAGE_CASE.INVESTIGATION_KEY = PAGE_CASE.INVESTIGATION_KEY------ (My table)
+                 INNER JOIN
+             #TMP_CONDITION AS T
+             ON TMP_F_PAGE_CASE.CONDITION_KEY = T.CONDITION_KEY------ (My table)
+                 LEFT OUTER JOIN
+             dbo.D_PROVIDER AS P WITH(NOLOCK)
+             ON PAGE_CASE.PHYSICIAN_KEY = P.PROVIDER_KEY
+                 LEFT OUTER JOIN
+             dbo.D_PROVIDER AS INVGTR WITH(NOLOCK)
+             ON PAGE_CASE.INVESTIGATOR_KEY = INVGTR.PROVIDER_KEY
+                 LEFT OUTER JOIN
+             dbo.D_ORGANIZATION AS REPTORG WITH(NOLOCK)
+             ON PAGE_CASE.ORG_AS_REPORTER_KEY = REPTORG.ORGANIZATION_KEY
+        ORDER BY HEP_PAT_PROV_INV_KEY;
+
+        UPDATE #TMP_HEP_PAT_PROV
+        SET PHYS_NAME = RTRIM(LTRIM(PHYSICIAN_first_nm));
+
+        UPDATE #TMP_HEP_PAT_PROV
```

Comment:
@upasanapattnaik-eq  could this be moved to the previous Select

Reply:
Will take the individual updates as enhancement items.